test/e2e-kind.sh

```diff
@@ -173,6 +173,31 @@ kubectl -n "${KOURIER_CONTROL_NAMESPACE}" patch configmap/config-kourier --type
 kubectl -n "${KOURIER_CONTROL_NAMESPACE}" rollout restart deployment/net-kourier-controller
 kubectl -n "${KOURIER_CONTROL_NAMESPACE}" rollout status deployment/net-kourier-controller --timeout=300s
 
+echo ">> Setup Tracing"
+kubectl apply -f test/config/tracing
+kubectl -n tracing wait --timeout=300s --for=condition=Available deployment/jaeger
+export TRACING_COLLECTOR_FULL_ENDPOINT="$(kubectl -n tracing get svc/jaeger -o jsonpath='{.spec.clusterIP}'):9411/api/v2/spans"
+kubectl -n "${KOURIER_CONTROL_NAMESPACE}" patch configmap/config-kourier --type merge -p "{
+  \"data\":{
+    \"tracing-collector-full-endpoint\": \"$TRACING_COLLECTOR_FULL_ENDPOINT\"
+  }
+}"
+kubectl -n "${KOURIER_CONTROL_NAMESPACE}" rollout restart deployment/net-kourier-controller
```

Comment:
The `rollout restart/status` should not be necessary as `config-kourier` change can be applied without restart.

We will remove other places like https://github.com/knative-sandbox/net-kourier/pull/1077

Reply:
It is not a new behavior. The controller watches the configmap which is stored in `./pkg/reconciler/ingress/config/store.go`  (`config-network` and `config-kourier`) via  https://github.com/knative-sandbox/net-kourier/blob/c8bcc9ae38e3b77270ebe48b0de8612d9a56b32a/pkg/reconciler/ingress/controller.go#L102-L107

Also IIRC, kourier reloads `conig-logging` and `config-observability` automatically thanks to the implementation of pkg's shared main.